xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamBluray.cpp

```diff
@@ -217,12 +217,23 @@ bool CDVDInputStreamBluray::Open()
       return false;
     }
   }
-  else
+  else if (!m_item.IsBluray())
   {
     m_rootPath = root;
     if (!bd_open_files(m_bd, &m_rootPath, CBlurayCallback::dir_open, CBlurayCallback::file_open))
     {
-      CLog::Log(LOGERROR, "CDVDInputStreamBluray::Open - failed to open %s", CURL::GetRedacted(root).c_str());
+      CLog::Log(LOGERROR, "CDVDInputStreamBluray::Open - failed to open %s in files mode",
+                CURL::GetRedacted(root).c_str());
+      return false;
+    }
+  }
+  else
+  {
+    m_rootPath = root;
+    if (!bd_open_disc(m_bd, CURL::GetRedacted(root).c_str(), nullptr))
```

Comment:
Why `GetRedacted` call is used here. It wasn't used before https://github.com/xbmc/xbmc/commit/25d755c0365b2699161bc303bd55afae895fc2ec#diff-276823fbbb9dfcd26bb50ffee871a7c3L383

Reply:
> Why `GetRedacted` call is used here. It wasn't used before [25d755c#diff-276823fbbb9dfcd26bb50ffee871a7c3L383](https://github.com/xbmc/xbmc/commit/25d755c0365b2699161bc303bd55afae895fc2ec#diff-276823fbbb9dfcd26bb50ffee871a7c3L383)
> It seems this method should be used only for logging purposes.

No reason, I didn't know it's only used for logging purposes. When I first experimented with this I just copy pasted it and forgat about it. But you are right, it isn't necessary and I've dropped it now.